### DISCUSSION TITLE
Displaying if a PR is in automerge mode in the title?

### DISCUSSION THREAD

**Discussion by laurent22** [renovatebot/renovate#19315] [Request Help] [answered]
Is it possible to add whether a PR is in automerge mode in the title? I see that for the `prTitle` they say this:

> The PR title is important for some of Renovate's matching algorithms (e.g. determining whether to recreate a PR or not) so ideally don't modify it much.

But what does it mean? Can I still add some info? And if so, what variable can I use for the automerge status?

Edit: Actually adding a label would be fine too (but not seeing this option). Basically anything that makes it clear that the PR is going to be auto merged, so there's no need to open it.

**Comment by rarkins** [comment, marked as answer]
You can configure `labels` in the same place you currently set `automerge=true`

**Comment by laurent22** [comment]
Right, I see it now, I was looking for the option in the wrong place. Thanks a lot for the info.
